phpstan.neon

```diff
@@ -12,10 +12,14 @@ parameters:
         SprykerSdk\Integrator\Transfer\AbstractTransfer:
             - throwNullValueException
 
+    excludePaths:
+        analyseAndScan:
+            - src/Transfer/AbstractTransfer.php
```

Comment:
I expect that this line will produce the questions. So I explain it. 

PHPStan has released a new patch version. And since that we get the new PHPStan error:

```
 ------ --------------------------------------------------------------------- 
  Line   Transfer/AbstractTransfer.php                                        
 ------ --------------------------------------------------------------------- 
  421    Return type mixed of method                                          
         SprykerSdk\Integrator\Transfer\AbstractTransfer::offsetGet() is not  
         covariant with tentative return type mixed of method                 
         ArrayAccess::offsetGet().                                            
         💡 Make it covariant, or use the #[\ReturnTypeWillChange] attribute  
         to temporarily suppress the error.                                   
 ------ --------------------------------------------------------------------- 
```
The problem is that it exists only for PHP 8.1 check. Method signature on PHP 8.1:

```
#[TentativeType]
public function offsetGet(#[LanguageLevelTypeAware(['8.0' => 'mixed'], default: '')] $offset): mixed;
```

We have only 1 version of code, that should work both on PHP 7.4, 8.0 and 8.1, so I can’t fix this PHPStan error by adding mixed return type or #[\ReturnTypeWillChange] to our code.

And I also can’t add this error to ignoreErrors in PHPStan configuration, because it will throw an exception on PHP 7.4 check: 

`Error message "..." cannot be ignored, use excludePaths instead.`

That's why as a solution, I exclude this file from the check.